### DISCUSSION TITLE
fastlane does not see new App Store Connect team

### DISCUSSION THREAD

**Discussion by valentin-at-edlio** [fastlane/fastlane#20534] [Q&A]
My Apple ID has access to quite a lot teams and recently I've joined 3 more.

fastlane shows this when it uses my Apple ID to login:
```
# iTunes client (App Store Connect)
The current user is in 25 teams. Pass a team ID or call `select_team` to choose a team. Using the first one for now.
# Portal client (Apple Developer Portal)
The current user is in 28 teams. Pass a team ID or call `select_team` to choose a team. Using the first one for now.
```

It sees the new teams on Developer Portal, but not on App Store Connect.
I've logged in via browser at https://appstoreconnect.apple.com/ and inspected the session (`https://appstoreconnect.apple.com/olympus/v1/session`) and the `availableProviders` array is indeed missing the new teams.

Funny thing is that the new teams are listed in the team selector dropdown (top-right corner of the site) and I am able to select them.

Do you know what is causing this? 
Is there any way to 'refresh' the session so it includes the new teams?

**Comment by SvetoslavTsenov** [comment]
Hey, team, I have the same issue. Do you have any advice?

`Could not set team ID to '******', only found the following available teams:`
There are a few new teams that are not listed by https://appstoreconnect.apple.com/olympus/v1/session but are available in the UI and all seems to be correct there

@valentin-at-edlio , did you find any solution?

**Reply by valentin-at-edlio** [reply]
The only workaround I have found so far is this:

1. Go to Apple Developer and create a new App Identifier
2. Go to App Store Connect and create an iOS App
3. Wait about an hour...

This is slow and complicated...

I have a team which has not been listed by https://appstoreconnect.apple.com/olympus/v1/session for a week now so simply waiting does not seem to help...

**Reply by SvetoslavTsenov** [reply]
Thanks @valentin-at-edlio, I will try it as well. Let's see how it goes.
